app/Http/Controllers/Api/ContestsController.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Http\Transformers\ContestTransformer;
+
+class ContestsController extends ApiController
+{
+    /**
+     * @var \Gladiator\Http\Transformers\UserTransformer
+     */
+    protected $transformer;
+
+    /**
+     * Create a new ContestsController instance.
+     *
+     * @param \Gladiator\Http\Transformers\ContestTransformer $transformer
+     */
+    public function __construct(ContestTransformer $transformer)
```

Comment:
I don't know if there's much value in injecting this from the service container (rather than setting `$this->transformer = new ContestTransformer`) since it doesn't have any dependencies of it's own.